roles/signals.py

```diff
@@ -45,6 +45,3 @@ def add_creator_permission(sender, **kwargs):
         instance,
         GroupTypes.REPO_ADMINISTRATOR,
     )
-    # This signal is only fired upon initial creation of a repository,
-    # so receivers don't have to check whether the repository is new.
-    repo_created.send(sender=instance)
```

Comment:
It's only used in creating default vocabularies so without that functionality there is no need for it. We can add it back in the PR implementing this functionality again